test/README.md

```diff
@@ -0,0 +1,27 @@
+## E2E Tests
+
+E2E tests are verifying successful deployment of the DevOps Console Operator and creation of self-provisioning service account.
```

Comment:
creation of self-provisioning account is functionality for toolchain operator. we should remove this.

Reply:
Fixed befe3ef.